pkg/cuedefs/config/config.cue

```diff
@@ -162,6 +167,22 @@ package config
 	keyPrefix: string | *"inngest:state"
 }
 
+// # DataStore
+//
+// DataStore stores the persisted data for the system.
+#DataStoreService: #Postgres //#InmemStore | #Postgres
```

Comment:
Do we need to enable the in-memory item here?

Reply:
Ah good - catch - I commented this out before I moved the `filesystem_loader.go` to `inmemory/inmemory.go` - Fixing in next commit.